README.md

```diff
@@ -73,7 +66,6 @@ uv pip install -e ".[dev]"
 [website]: https://github-pages.arc.ucl.ac.uk/python-tooling
 [UCL ARC]: https://ucl.ac.uk/arc
 [cookiecutter]: https://cookiecutter.readthedocs.io/en/stable
-[install cookiecutter]: https://cookiecutter.readthedocs.io/en/stable/README.html#installation
```

Comment:
We either need this. Or remove the link in the text body :point_up: .

I'd maybe keep this as is but delete the `using pip,` (since I have no preference how you install `cookiecutter` but probably `cookiecutter`'s docs do.